.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "*.js": "eslint . --fix"
+}
```

Comment:
This would lint all js files in the project, not just the staged ones?

Reply:
Yes, you are right! Just fixed